x/observer/keeper/pending_nonces_test.go

```diff
@@ -136,4 +137,54 @@ func TestKeeper_RemoveFromPendingNonces(t *testing.T) {
 		}
 		require.True(t, nonceUpdated)
 	})
+
+	t.Run("test removal within range only using fixed value", func(t *testing.T) {
+		k, ctx, _, _ := keepertest.ObserverKeeper(t)
+
+		tss := sample.Tss()
+		// make nonces and pubkey deterministic for test
+		chainIDS := []int64{chains.GoerliLocalnet.ChainId, chains.BitcoinTestnet.ChainId, chains.BscTestnet.ChainId}
+		pn := make([]types.PendingNonces, len(chainIDS))
```

Comment:
```suggestion
		pendingNonces := make([]types.PendingNonces, len(chainIDS))
```
better to keep explicit variable names